CAIRN Info.js

```diff
@@ -84,6 +84,22 @@ function doWeb(doc,url)
 						tags.push(keywords[i])
 					}
 				}
+				
+				// The default value for PDF download is on an HTML page that calls the actual download
+				// we need to add this attachment after the import translator has been running
+				// We remove the attached PDF because they won't be working
+				for(var i=0, n=item.attachments.length; i<n; i++) {
```

Comment:
would we ever have two PDFs? because then we'd want to have the for loop count down from attachments.length when using delete, right?